apps/server/src/modules/learnroom/learnroom-api.module.ts

```diff
@@ -3,7 +3,7 @@ import { AuthorizationReferenceModule } from '@modules/authorization/authorizati
 import { CopyHelperModule } from '@modules/copy-helper';
 import { LessonModule } from '@modules/lesson';
 import { Module } from '@nestjs/common';
-import { CourseRepo, DashboardModelMapper, DashboardRepo, LegacyBoardRepo, UserRepo } from '@shared/repo';
+import { CourseRepo, DashboardModelMapper, DashboardRepo, LegacyBoardRepo, RoleRepo, UserRepo } from '@shared/repo';
```

Comment:
We should import the `RoleModule` into the `LearnRoomApiModule` and use the `RoleService` from there instead of importing and providing `RoleService` and `RoleRepo` directly. `RoleService` belongs to the public interface of `RoleModule` (as well as `RoleRepo` - but that might not be necessary).